lib/ContentHelpers.php

```diff
@@ -149,10 +149,21 @@ public static function extractFlexibleContent(Entry $entry, $locale, $children =
         foreach ($entry->flexibleContent->all() as $block) {
             switch ($block->type->handle) {
                 case 'contentArea':
+
+                    // Scale down any in-body images
+                    $contentWithResizedImages = preg_replace_callback(
+                        '/<img src="(https:\/\/www.tnlcommunityfund.org.uk\/media\/.+?)"/i',
+                        function ($matches) {
+                            $url = Images::imgixUrl($matches[1], ['w' => 900, 'fit' => 'fill']);
```

Comment:
Does something like `max-w` or https://docs.imgix.com/apis/url/size/fit#max work, i.e. say that an image can't be _bigger_ than a certain limit?